### ISSUE TITLE
Friends to main schema?

### ISSUE THREAD

**Issue by addiinnocent** [toplocs/tribelike#41]
Should we add the functionalities regarding friends (add, remove, suggestions etc.) to the main product or should it become a plugin?

**Comment by felixboehm** [comment]
Yes, let's move this to a plugin. We should have a tab "Members" anyway ...

**Comment by felixboehm** [comment]
We need to understand the use case better. What is the value for adding friendship?
